scripts/o3de/tests/ExportScripts/test_export_source_built_project.py

```diff
@@ -0,0 +1,1351 @@
+        mock_ctx.project_path = test_project_path
+        mock_ctx.engine_path = test_engine_path
+        mock_ctx.project_name = test_project_name
+
+        mock_platform = 'pc'
+
+        for should_build_tools, toolconf in zip([True, False, True, False],
+                                                ['profile', 'profile', 'release', 'release']):
+            
+            for base_path in [None, test_o3de_base_path, test_absolute_base_path, test_relative_base_path]:
+                
+                for launcher_indices in launcher_powerset_indices():
+                    launcher_set = set(launcher_indices)
+                    check_launcher_type = 0
+
+                    should_build_game  = 1 in launcher_set
+                    should_build_server = 2 in launcher_set
+                    should_build_unified = 3 in launcher_set
+
+                    if should_build_game:
+                        check_launcher_type |= LauncherType.GAME
+                    if should_build_server:
+                        check_launcher_type |= LauncherType.SERVER
+                    if should_build_unified:
+                        check_launcher_type |= LauncherType.UNIFIED
+
+                    test_tools_build_path = None if not base_path else base_path / 'build/tools'
+                    test_tools_sdk_path = (test_engine_path / 'bin/Windows' / toolconf / 'Default')
+
+                    test_launcher_build_path = None if not base_path else base_path / 'build/launcher'
+
+                    selected_tools_build_path = test_tools_build_path if not use_sdk else test_tools_sdk_path
+                    selected_launcher_build_path = test_launcher_build_path
+
+                    if not selected_tools_build_path:
+                        selected_tools_build_path = test_o3de_base_path / 'build/tools'
+                    
+                    if not selected_tools_build_path.is_absolute():
+                        selected_tools_build_path = test_o3de_base_path / selected_tools_build_path
+                    
+                    if not selected_launcher_build_path:
+                        selected_launcher_build_path = test_o3de_base_path / 'build/launcher'
+
+                    if not selected_launcher_build_path.is_absolute():
+                        selected_launcher_build_path = test_o3de_base_path / selected_launcher_build_path
+
+                    cannot_build_monolithic = (use_sdk and not has_monolithic and use_monolithic)
+
+                    
+                    
+                    if cannot_build_monolithic:
+                        pytest.raises(exp.ExportProjectError, export_standalone_project,
+                                      mock_ctx,
+                                      mock_platform,
+                                      test_output_path,
+                                      should_build_tools, 
+                                      "release",
+                                      toolconf,
```

Comment:
isn't it the tools that must always be "profile" and the launcher can be "profile" or "release"?